pocs/camera/sbigudrv.py

```diff
@@ -8,20 +8,15 @@
 ctypes.Structures, plus a class (SBIGDriver) to load the library
 and call the single command function (SBIGDriver._send_command()).
 """
-import platform
 import ctypes
 from ctypes.util import find_library
 from warnings import warn
-import _ctypes
```

Comment:
@AnthonyHorton is there any chance this is doing some magic simply by importing?